miso-web/src/main/webapp/scripts/form_run.js

```diff
@@ -205,6 +207,47 @@ FormTarget.run = (function($) {
           title: 'Completion Date',
           data: 'endDate',
           type: 'date'
+        }, {
+          title: 'Data Approved',
+          data: 'dataApproved',
+          type: 'dropdown',
+          getSource: function() {
+            return [{
+              label: 'True',
+              value: true
+            }, {
+              label: 'False',
+              value: false
+            }];
+          },
+          convertToBoolean: true,
```

Comment:
What do you think about `Approved`/`Not Approved`?

Reply:
For the dropdown labels you mean? "Data Approved: Not Approved" seems kind-of weird

Maybe "Yes"/"No" would be better than "True"/"False"?

Reply:
I like "Yes"/"No"!